api/RS.API/RS.Web/Controllers/CandidateController.cs

```diff
@@ -82,5 +102,47 @@ public IResult ApprovedForInterview(Guid candidateId)
             var approvedCandidate = _candidateManagerService.ApprovedForInterview(candidateId);
             return approvedCandidate;
         }
+
+        private string GetExtension(IFormFile file)
```

Comment:
create common methods for file handling under common layer.

Reply:
Done